eos-paygd/com.endlessm.Payg1.service.in

```diff
@@ -1,5 +1,5 @@
 [D-BUS Service]
```

Comment:
If your service is now launched by the initramfs, we can't trust the one that would be dbus activated on the main root. So perhaps all this dbus activation stuff should be dropped?

Reply:
Yes let's delete that. I doubt it would work anyway with `BusName=` removed from `eos-paygd.service` and as you say we don't want it to work from the root fs.